lib/commands/samples.ts

```diff
@@ -49,3 +49,39 @@ class CloneCommandParameter implements ICommandParameter {
 		}).future<boolean>()();
 	}
 }
+
+class PrintCordovaSamplesInformationCommand implements ICommand {
```

Comment:
I've tried using BaseCommand for all of them and use constructor injection for projectConstants and samplesService in it, but in PrintCordovaSamplesInformationCommand I'll have to inject both of them in order to call super(...):
```TypeScript
class PrintCordovaSamplesInformationCommand extends PrintBaseCommand {
	constructor(protected $samplesService: ISamplesService,
		protected $projectConstants: Project.IProjectConstants) {
		super($samplesService, $projectConstants);
	}
```
So we still have much of the code duplicated. In this case I prefer to keep the current code.